debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorTask.java

```diff
@@ -103,6 +103,8 @@ public ChangeEventSourceCoordinator start(Configuration config) {
                     metadataProvider,
                     schemaNameAdjuster);
 
+            dispatcher.setNeverSkip(true);
```

Comment:
From a DSL perspective I find an approach that I sketched in the other comment `dispatcher.disableOperationSkipping()` better. WDYT?